redis/mutation_utils.cc

```diff
@@ -68,6 +68,11 @@ future<> write_strings(service::storage_proxy& proxy, redis::redis_options& opti
     return proxy.mutate(std::vector<mutation> {std::move(m)}, write_consistency_level, timeout, nullptr, permit);
 }
 
+future<> write_strings_pairs(service::storage_proxy& proxy, redis::redis_options& options, std::map<bytes, bytes>&& data_map, long ttl, service_permit permit) {
+	return parallel_for_each(data_map.begin(), data_map.end(), [&proxy, &options, ttl, permit] (auto &pair) {
+		return write_strings(proxy, options, to_bytes(pair.first), to_bytes(pair.second), ttl, permit);
```

Comment:
Why is this `to_bytes()` needed, aren't both elements of the pair of type `bytes`?